Reporting/M5 Document/StepByStep.md

```diff
@@ -0,0 +1,97 @@
+ -   Getting started:
+
+        -   If this is the first time you are pulling from the repo,
+            > open a new command line and run npm install (make sure you
+            > install Node.js and the npm package manager first if you
+            > don\'t have this on your system).
+
+        -   To start up the server, run npm run dev.
+
+        -   As per the instructions from the terminal, navigate to
+            > [[http://localhost:3000/]{.underline}](http://localhost:3000/).
+
+        -   Next.js is a React framework that adds additional
+            > functionality on top of what React already has to offer.
+            > Its main selling point is server-side rendering. During
+            > the development process, we will be running the
+            > development server to work on our web application. This is
+            > recommended for development. However, once our project is
+            > completed, we will build the pages for our application
+            > using npm build to demo our application in production
+            > mode.
+
+        -   The main sections we will be working from are the app and
+            > components directory. SInce we are using React, all of our
+            > components will exist in our components directory and will
+            > be imported into the respective pages within the app
+            > directory. The layout.tsx file is just a general layout
+            > file that has already been set up and does not need to be
+            > modified until further notice. For example, the page.tsx
+            > file is what you currently see when you launch the
+            > application.
+
+    -   MongoDB Atlass:
+
+        -   Go to MongoDB Atlas and create an account
```

Comment:
Omit this part. This is for us only. MongoDB is whitelisted to all IP addresses so we don't need to give users access to our backend.